<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Base</class>
 <widget class="QWidget" name="Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base</string>
  </property>
  <widget class="QPushButton" name="rs_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run script</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="domain_in">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="target_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Domain</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="log_out">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>541</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="graph_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph show</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
